.github/workflows/test_and_coverage.yml

```diff
@@ -0,0 +1,41 @@
+name: Python tox + coverage CI
+
+# You may need to add pull_request for the coverall action
+#on: ["push", "pull_request"]
+on: "push"
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install Dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install tox tox-gh-actions
+        pip install -r example/requirements.txt
+    - name: Run Tox
+      run: tox
+    - name: Report to coverall  # by the package as the action is broken, but won't work on external forks!
+      if: matrix.python-version == 3.7
+      env:
+        COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
+      run: |
+        pip install coveralls
```

Comment:
I wouldn't mind removing coveralls completely and switch to a better integrated solution like codecov in the future.

Reply:
commented out for now